charts/datadog/README.md

```diff
@@ -486,12 +484,12 @@ helm install --name <RELEASE_NAME> \
 | datadog.site | string | `nil` | The site of the Datadog intake to send Agent data to |
 | datadog.systemProbe.apparmor | string | `"unconfined"` | Specify a apparmor profile for system-probe |
 | datadog.systemProbe.bpfDebug | bool | `false` | Enable logging for kernel debug |
-| datadog.systemProbe.collectDNSStats | bool | `false` | Enable DNS stat collection |
+| datadog.systemProbe.container_scrubbing.enabled | bool | `true` |  |
 | datadog.systemProbe.debugPort | int | `0` | Specify the port to expose pprof and expvar for system-probe agent |
 | datadog.systemProbe.enableConntrack | bool | `true` | Enable the system-probe agent to connect to the netlink/conntrack subsystem to add NAT information to connection data |
 | datadog.systemProbe.enableOOMKill | bool | `false` | Enable the OOM kill eBPF-based check |
 | datadog.systemProbe.enableTCPQueueLength | bool | `false` | Enable the TCP queue length eBPF-based check |
-| datadog.systemProbe.enabled | bool | `false` | Set this to true to enable system-probe agent |
+| datadog.systemProbe.enabled | bool | `false` | DEPRECATED. Set this to true to enable system-probe agent. Prefer using `networkMonitoring.enabled`, `systemProbe.enableTCPQueueLength`, `systemProbe.enableOOMKill`, or `securityAgent.runtime.enabled` to enable specific features. |
```

Comment:
I didn't see other examples of us deprecating config options, so I'm not sure if this is a good pattern or if I should simply delete this block entirely.

Reply:
You could add a deprecation message in the `NOTES.txt`. You can find some example in this file: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/templates/NOTES.txt#L139